osier/technology.py

```diff
@@ -201,9 +202,13 @@ class Technology(object):
         Default is 1.0, i.e. all of the technology's capacity contributes
         to capacity requirements.
     co2_rate : float or :class:`unyt.array.unyt_quantity`
-        Specifies the rate at carbon is emitted. May be either lifecycle
-        emissions or from direct use. However, consistency between
-        technologies is incumbent on the user.
+        Specifies the rate carbon dioxide is emitted during operation.
```

Comment:
```suggestion
        Specifies the rate at which carbon dioxide is emitted during operation.
```
Also, what units are these in? Is there a parameter to modify it? I assume it's the same for the units for `co2_rate`...

Reply:
Yes, it is modifiable using lifecycle_co2_rate